src/Main/Main.scss

```diff
@@ -0,0 +1,38 @@
+$primary: #17A2B8;
+
+body {
+    min-width: 370px;
+}
+
+.title-box-width {
+    max-width: 100%;
+    width: 650px;
+    height: 200px;
+}
+
+.banner-title {
+    font-size: 50px;
+}
```

Comment:
this belongs to the Title component, so it should be in a separate css file for that component

Reply:
scss i mean